src/tools/vmap4_extractor/wdtfile.cpp

```diff
@@ -113,6 +113,6 @@ ADTFile* WDTFile::GetMap(int x, int z)
 
     char name[512];
 
-    sprintf(name, "World\\Maps\\%s\\%s_%d_%d.adt", filename.c_str(), filename.c_str(), x, z);
+    sprintf(name, R"(World\Maps\%s\%s_%d_%d.adt)", filename.c_str(), filename.c_str(), x, z);
```

Comment:
The same should be done here. Codefactor is afraid that the buffer might overflow:
_Does not check for buffer overflows (CWE-120)._
[Top 25 most dangerous software weaknesses](https://cwe.mitre.org/data/definitions/120.html)